config.json

```diff
@@ -12,11 +12,13 @@
         "port": 587
     },
     "api_config": {
+        "dev_port": 5000,
         "MAX_USERINFO_REQUEST": 100,
         "MAX_USERID_REQUEST": 10000,
         "MAX_ARTICLEDATA_REQUEST": 100,
         "MAX_RECOMMENDATION_USERS": 100,
-        "MAX_RECOMMENDATION_ARTICLES": 10
+        "MAX_RECOMMENDATION_ARTICLES": 10,
+        "MAX_EXPLANATION_LEN": 100
```

Comment:
Make sure this limit is mentioned in the [API documentation](https://github.com/iai-group/arXivDigest/tree/master/api)